index.html

```diff
@@ -40,55 +40,83 @@
       <hr class="underline">
       <div class="card-container">
         <div class="card">
-          <img src="images/conference-icon.png" alt="Conference Icon">
+          <img class="card-icon" src="images/conference-icon.png" alt="Conference Icon">
           <h5 class="card-title">Lecture</h5>
-          <p class="card-text">Listen to the spekers from various countries about the messages of sharing and opening
-          </p>
+          <div class="card-text-container">
+            <p class="card-text">Listen to the spekers from various countries about the messages of sharing and opening.</p>
+          </div>
         </div>
         <div class="card">
-          <img src="images/exhibition-icon.png" alt="Exhibition Icon">
+          <img class="card-icon" src="images/exhibition-icon.png" alt="Exhibition Icon">
           <h5 class="card-title">CC exhibition</h5>
-          <p class="card-text">Let's meet the creations of artists from various fields who share the open spirit of CC.
-          </p>
+          <div class="card-text-container">
+            <p class="card-text">Let's meet the creations of artists from various fields who share the open spirit of CC.</p>
+          </div>
         </div>
         <div class="card">
-          <img src="images/forum-icon.png" alt="Forum Icon">
+          <img class="card-icon" src="images/forum-icon.png" alt="Forum Icon">
           <h5 class="card-title">Forum</h5>
-          <p class="card-text">We have time to share our thoughts and opinions with experts by topic.</p>
+          <div class="card-text-container">
+            <p class="card-text">We have time to share our thoughts and opinions with experts by topic.</p>
+          </div>
         </div>
         <div class="card">
-          <img src="images/workshop-icon.png" alt="Workshop Icon">
+          <img class="card-icon" src="images/workshop-icon.png" alt="Workshop Icon">
           <h5 class="card-title">Workshop</h5>
-          <p class="card-text">You can create your own creations using open source instead of just looking at them.</p>
+          <div class="card-text-container">
+            <p class="card-text">You can create your own creations using open source instead of just looking at them.</p>
+          </div>
         </div>
         <div class="card">
-          <img src="images/party-icon.png" alt="Party Icon">
+          <img class="card-icon" src="images/party-icon.png" alt="Party Icon">
           <h5 class="card-title">CC party</h5>
-          <p class="card-text">Create opportunities to freely network with CC personnel from around the world.</p>
+          <div class="card-text-container">
+            <p class="card-text">Create opportunities to freely network with CC personnel from around the world.</p>
+          </div>
         </div>
-        <a href="#">View full program></a>
+      </div>
+      <a class="view" id="view" href="#">View full program></a>
+    </section>
```

Comment:
- [ ] In the main program section, the cards are supposed to be arranged in a column on the desktop version and not in a row. Please kindly fix that.
- [ ] On the mobile version I can see that the page is overflowing. Please kindly fix that.

Reply:
The cards are showing as on the design. Screenshot attached.
![comparison](https://user-images.githubusercontent.com/53036416/143319705-d0ef14dc-27ae-470a-816d-2cbf046b560f.png)